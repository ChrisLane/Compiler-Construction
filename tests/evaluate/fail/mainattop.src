main() {
    print 1;
}

fun2() {
    print 2;
}
