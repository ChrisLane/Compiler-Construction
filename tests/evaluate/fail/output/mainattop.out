Syntax error at line 5 and column 0: fun2.
Fatal error: exception Error.Error
